x-pack/test/functional/apps/lens/group3/gauge.ts

```diff
@@ -48,6 +48,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     it('should reflect edits for gauge', async () => {
+      await PageObjects.lens.switchToVisualization('horizontalBullet', 'gauge');
```

Comment:
What are these changes about? It should work just fine without this change, right?

Reply:
If you'll run the test via `grep` by picking the current test case, it will fail. According to the FIRST principle of testing, a test should set up its own data and should not depend on any external factors to run its test. So, that is not really the right behavior, if the test is failing if ran separately from the previous test case. WDYT?